python/ray/autoscaler/_private/monitor.py

```diff
@@ -252,6 +252,9 @@ def update_load_metrics(self):
         response = self.gcs_node_resources_stub.GetAllResourceUsage(
             request, timeout=60)
         resources_batch_data = response.resource_usage_data
+        logger.debug(">>>Logging raw resource data.")
+        logger.debug(resources_batch_data)
+        logger.debug(">>>Done logging raw resource data.")
```

Comment:
Will tell you everything the autoscaler knows about resource usage in the cluster.